### ISSUE TITLE
Tab Groups missing tabs issue [ & a possible suggested workaround ]

### ISSUE THREAD

**Issue by wedebe** [machinateur/android-chrome-tab-transfer#29]
I noticed a few tabs were missing from my transferred list. I've got a bunch of tab groups (yuk!), so figured that may be the root cause. My workaround was to set a chrome flag, after which the missing tabs (which I'd noticed, at least) were successfully transferred :)

Suggestion: Add a note in the readme that changing `Tab Group Parity Android` ( chrome://flags/#tab-group-parity-android ) on the device to **Enabled** and relaunching Chrome may help.

Perhaps other users could verify whether this works for them?

**Comment by machinateur** [comment]
Hey, thanks for reporting this.

I have collected some tab groups by now on my new phone, but whenever I came back to this and tried it, I never had any missing tabs.

When I had something like this in the past, it usually happened at random and on the next try it worked perfectly again. I will note down to add this information as an edge-case scenario to the README. Until then I'll leave this issue open for informational purposes.

---

From what I could find online, I can't really say if the flag would be a general way of fixing this. But it's worth mentioning a possible fix exists for such cases.

> https://chromium.googlesource.com/chromium/src/+/a3721ff074c098d7b516dc672ec5ada5ef9ccad9/chrome/browser/flag_descriptions.cc#4841
